docs/cases/cases-manage-settings.asciidoc

```diff
@@ -1,10 +1,34 @@
-[[cases-ui-integrations]]
-== Configure external connections
-:frontmatter-description: Create and add external connectors to send cases to third-party systems.
+[[cases-manage-settings]]
+== Configure case settings
+:frontmatter-description: Change the default behavior of cases by adding connectors, custom fields, templates, and closure options.
 :frontmatter-tags-products: [security] 
 :frontmatter-tags-content-type: [how-to] 
 :frontmatter-tags-user-goals: [analyze]
 
+To change case closure options and add custom fields, templates, and connectors for external incident management systems, go to *Cases* -> *Settings*.
+
+[role="screenshot"]
+image::images/cases-settings.png[Shows the case settings page]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+* <<close-sent-cases,Set case closure options>>.
+* <<cases-ui-integrations,Add connectors for external incident management systems>>.
+* <<cases-ui-custom-fields,Add custom fields>>.
+* <<cases-templates,Add templates>>.
+
+[[close-connector]]
```

Comment:
Sorry, just noticed that this might be a left-over anchor.

Reply:
I don't see these anchors used in the Kibana doc link service, but just in case they're used elsewhere and need to be re-added, I'll defer removing them to a separate PR.